<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="TaurusLabel" name="stateLabel"/>
   </item>
   <item>
    <widget class="QPushButton" name="configButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="preferences-system"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TaurusLCD" name="positionLCD">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="unitLabel">
     <property name="text">
      <string>unit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TaurusValueLineEdit" name="absMotionEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="goButton">
     <property name="text">
      <string>Go</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-start"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="moveNegButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="media-seek-backward"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TaurusWheelEdit" name="relativeWheel">
     <property name="showArrowButtons">
      <bool>true</bool>
     </property>
     <property name="integerDigits" stdset="0">
      <number>4</number>
     </property>
     <property name="decimalDigits" stdset="0">
      <number>4</number>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
     <property name="minValue">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="movePosButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="media-seek-forward"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="disableButton">
     <property name="text">
      <string>Disable</string>
     </property>
     <property name="icon">
      <iconset theme="emblem-unreadable"/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusWheelEdit</class>
   <extends>QFrame</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
  <customwidget>
   <class>TaurusLabel</class>
   <extends>QLabel</extends>
   <header>taurus.qt.qtgui.display</header>
  </customwidget>
  <customwidget>
   <class>TaurusValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>taurus.qt.qtgui.input</header>
  </customwidget>
  <customwidget>
   <class>TaurusLCD</class>
   <extends>QLCDNumber</extends>
   <header>taurus.qt.qtgui.display</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
